src/ToggleButton/ToggleButton.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react';
+
+export class ToggleButton extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      className: props.className || '',
+      text: props.text || 'Button',
+      toggleText: props.toggleText || props.text || 'Button'
+    }
+  }
+
+  clickHandler = () => {
+    const { className, text, toggleText } = this.state;
+    const { toggleComponentId } = this.props;
+
+    const elem = document.getElementById(toggleComponentId);
+
+    if (/active/i.test(className)) {
```

Comment:
do not use reg exp for such simple validation (instead you can use includes method)